### PR TITLE
Add upload conda

### PR DESCRIPTION
**Pull request by bnemanich**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [x] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

This PR adds a workflow that will upload the `open-ce` conda package to Anaconda.org upon a release being published.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: open_ce/__init__.py [hunk 1]
```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 # *****************************************************************
 """
-__version__ = "1.1.3"
+__version__ = "2.0.0"
```
Comment (npanpaliya) [inline open_ce/__init__.py:18]: Should it be 1.2.0 or 2.0.0? This corresponds to the open-ce release.

Comment (bnemanich) [reply open_ce/__init__.py:18]: The build tools will be on a different release cadence than Open-CE. The next release of the build tools will be 2.0, whereas the next release of Open-CE will be 1.2.

**Review by jayfurmanek — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -41,21 +41,24 @@ When building packages that use CUDA, a tar package of TensorRT for the intended
 
 ### Installing the Open-CE Build Tools
 
-To get the Open-CE build tools, one can either clone the source code from [github](https://github.com/open-ce/open-ce-builder) or install it via `pip` as below - 
+To get the Open-CE build tools, one can either install it via `conda`, via `pip` or clone the source code from [github](https://github.com/open-ce/open-ce-builder) as below - 
 
+```bash
+# Conda install from the open-ce channel
+conda install -c open-ce open-ce
```
Comment (bethune-bryant) [inline README.md:48]: Should the channel and package here have -builder?
At least the package should probably.

**Review by bethune-bryant — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bnemanich) [reply README.md]: Yep, thanks!

**Review by bethune-bryant — APPROVED** [review]
LGTM

**Pull request merged**